OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -132,7 +132,7 @@ public SquadManagerBotModule(Actor self, SquadManagerBotModuleInfo info)
 		// Use for proactive targeting.
 		public bool IsPreferredEnemyUnit(Actor a)
 		{
-			if (a == null || a.IsDead || Player.RelationshipWith(a.Owner) != PlayerRelationship.Enemy || a.Info.HasTraitInfo<HuskInfo>())
+			if (a == null || a.IsDead || Player.RelationshipWith(a.Owner) != PlayerRelationship.Enemy || a.Info.HasTraitInfo<HuskInfo>() || a.Info.HasTraitInfo<AircraftInfo>())
```

Comment:
This one is a little bit incompatible with some mods that has slow and hover aircraft. In doing so player's air unit will be ignored in air/ground squad target selection.

To solve d2k problem we should give the carryall a target type then ignore it in `IgnoredEnemyTargetTypes` instead of hardcode it here.